### PR TITLE
Login UX + Change default route "/" to index/browse component

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

**Comment by xbill82** [comment]
The PR is LGTM, but I think we should provide a better error reporting on login failure. Currently, users won't know whether the login failed because Kuzzle is broken or because the provided credentials are bad.

But this should be a separate ticket.

**Comment by stafyniaksacha** [comment]
Yep, I agree with your remark, but I think we have to dig into an error reporting system separately

File: src/components/Login.vue [hunk 1]
```diff
@@ -1,42 +1,66 @@
...
-          <div class="row">
-            <div class="col m6">
-              <p class="error">{{error}}</p>
-            </div>
-            <div class="col m6">
-              <p class="right-align">
-                <button class="btn waves-effect waves-light" type="submit" name="action">Login</button>
-              </p>
-            </div>
+            <form class="col s10 offset-s1" id="loginForm" method="post" @submit.prevent="login(username, password)">
```
Comment (AnthonySendra) [inline src/components/Login.vue]: Why do we send `username` and `password` in parameters? We have directly access in method with `this.username` thanks to `v-model`

Comment (stafyniaksacha) [inline src/components/Login.vue]: ok for feedback; will be implemented into the #55

**Comment by AnthonySendra** [comment]
:+1:

**Comment by jenow** [comment]
+1

**Pull request merged**